app/views/groups/show.html.slim

```diff
@@ -64,9 +64,12 @@
         - @group.tasks.each do |task|
           .table-list
             .btn-group
-              = link_to task.title, task_path(task), class: 'btn btn-light', style: "border-right: none"
-              = link_to remove_task_group_path(task: task), method: :patch, class: 'btn btn-light', style: "border-left: none; margin-right: 5px;"  do
-                i.fa-solid.fa-xmark style=("color:gray;")
+              - if policy(@group).remove_task?
+                = link_to task.title, task_path(task), class: 'btn btn-light', style: "border-right: none"
+                = link_to remove_task_group_path(task: task), method: :patch, class: 'btn btn-light', style: "border-left: none; margin-right: 5px;"  do
```

Comment:
I would remove the whole custom styling for the remove-button, because I actually like the divider there:

```suggestion
                = link_to remove_task_group_path(task: task), method: :patch, class: 'btn btn-light' do
```

![image](https://github.com/openHPI/codeharbor/assets/24592181/3fed8f10-abc6-4ef8-9651-69523cf6ccfa)